vulnerabilities/data_dump.py

```diff
@@ -68,3 +70,55 @@ def ubuntu_dump(html):
             vulnerability=vulnerability,
             package=package
         )
+
+
+def archlinux_dump(extract_data):
+    """
+    Save data scraped from archlinux' security tracker.
+    Args:
+        extract_data(generator): data collected from archlinux' security tracker
+    """
+    for data in extract_data:
+        packages_name = data['packages_name']
+        vulnerabilities = data['vulnerability_id']
+        affected_version = data['version'][0]
+        fixed_version = data['version'][1]
+
+        vulnerability = Vulnerability.objects.create(
+            summary=data['description'],
+        )
+
+        for vulnerability_id in vulnerabilities:
+            VulnerabilityReference.objects.create(
+                vulnerability=vulnerability,
+                reference_id=vulnerability_id,
+                source='archlinux',
```

Comment:
Not all is from archlinux... CVEs are not from there.
And we should have a URL for each https://github.com/nexB/vulnerablecode/blob/55a633db3c09d8b78d21bffea25a510e1444acdf/vulnerabilities/models.py#L48